src/commands/cluster-keyslot.json

```diff
@@ -4,16 +4,36 @@
         "complexity": "O(N) where N is the number of bytes in the key",
         "group": "cluster",
         "since": "3.0.0",
-        "arity": 3,
+        "arity": -3,
         "container": "CLUSTER",
         "function": "clusterCommand",
         "command_flags": [
-            "STALE"
+            "READONLY"
+        ],
+	"key_specs": [
+            {
+                "flags": [
+                    "RO"
```

Comment:
i'm not sure RO is right.
considering that it doesn't open the key at all.
on the other hand the docs say:

> Every key specification has precisely one of the following flags

maybe that's an indication that this whole PR is wrong, i.e. that command takes a key name, but it doesn't work on that key at all.

in fact, i suppose that the command is meant to be executed on any cluster node in order to know which slot (and which node) that key belongs to.
and as soon as we add that data, a cluster aware client library may decide to always send the command to that one node.

please note that adding this data will also cause the [firstkey, lastkey, keystep] triplet to be available.

@madolson @guybe7 feel free to comment.

Reply:
I agree that think this PR is wrong, these aren't "Keys" like `MEMORY USAGE` accesses keys, so I believe the current code is correct.

Reply:
regarding COMMAND GETKEYS, it dpes returns the list of keys the commands work on and it does indeed work on sub-commands.
but this sub-command (CLUSTER KEYSLOT) doesn't "work" on keys, and adding a [first,last,step] or key-specs to its COMMAND INFO would be wrong.

Reply:
Sure, Thank you for clarifying it.